src/Storefront/Resources/snippet/en_GB/storefront.en-GB.json

```
@@ -806,7 +806,13 @@
     "groupComfortFeaturesYoutubeVideo": "YouTube video",
     "groupMarketing": "Marketing",
     "groupMarketingDescription": "Allows Google to collect personal data for online advertising and marketing.",
-    "groupMarketingAdConsent": "Google Advertising"
+    "groupMarketingAdConsent": "Google Advertising",
+    "wishlist" : {
+      "description": "In order to use the wishlist functionality you need to accept the \"Wishlist\" cookie or log in to your account.",
+      "title": "Missing cookie",
```

```suggestion
      "title": "Required cookie missing",
```
I'd suggest that, to make it clearer.